language/compiler/ir-to-bytecode/syntax/src/lexer.rs

```diff
@@ -123,7 +123,7 @@ impl Tok {
 
 pub struct Lexer<'input> {
     pub spec_mode: bool,
-    file: &'static str,
+    file: &'input str,
```

Comment:
I would just have this be a `Symbol` or `Atom` or whatever we end up doing

Reply:
Done in the latest commit!

Reply:
Looks like it is still a `&'input str` here. Am I looking at the wrong commit?

Reply:
Or do you mean on a local commit?